django_rq/management/commands/rqenqueue.py

```diff
@@ -22,9 +22,7 @@ def handle(self, *args, **options):
         Queues the function given with the first argument with the
         parameters given with the rest of the argument list.
         """
-        verbosity = int(options.get('verbosity', 1))
```

Comment:
These will all exist, and using indexed access will tell mypy they are not `None`